queries/unix_heartbeat.kusto

```diff
@@ -2,6 +2,14 @@
 // Display the VM's reported availability during the last 5 minutes.
 Heartbeat
 | extend affected_object = toupper(tostring(split(_ResourceId, "/") [-1]))
+| join kind= leftouter (MonitoringResources_CL 
+| where TimeGenerated > ago(13h)
+| where type_s == "microsoft.compute/virtualmachines" 
+| extend vm_MonitoringResources = toupper(tostring(split(id_s, "/") [-1])) 
+| project TimeGenerated,vm_MonitoringResources, id_s, name_s, tags_managedby_s
+| summarize arg_max(TimeGenerated,*) by vm_MonitoringResources)
+on $left.affected_object == $right.vm_MonitoringResources
+| where tolower(tags_managedby_s) == "q.beyond"
```

Comment:
```suggestion
| where tags_managedby_s =~ "q.beyond"
```
There is a case insensitive match operator that can be used here. Not really relevant for this tag but can be convenient when the target tag contains a mix capital and lower case letters